system-addon/content-src/components/LinkMenu/LinkMenu.jsx

```diff
@@ -0,0 +1,45 @@
+const React = require("react");
+const ContextMenu = require("content-src/components/ContextMenu/ContextMenu");
+
+class LinkMenu extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {visible: false};
+    this.getOptions = this.getOptions.bind(this);
+  }
+  getOptions() {
```

Comment:
Eventually this will have logic with bookmarking and with pocket and dismiss/delete for first run sites and blah blah blah so it's probably better to put it in a function. That's how the original code is, afterall